simulariumio/data_objects/file_data.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+
+from ..exceptions import DataError
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class FileData:
+    file_path: str
+    file_contents: str
+
+    def __init__(
+        self,
+        file_path: str = "",
+        file_contents: str = "",
+    ):
+        """
+        This object contains info about a file of data
```

Comment:
Hmmmm........ I wonder if `DataFile` would make more sense than `FileData`... `FileData` sounds more like something that `get_data()` would return. Or maybe `InputFile`? `ConverterInput`?

Reply:
Yeah, I like `InputFile`. Or `InputFileData`. But `FileData` also seems fine to me. I do agree this comment could be clearer, now that you've pointed it out.  It seems like a doc string for the whole class would be much more helpful than for just the init function, something like:  "This class handles the input file information, it contains one of two pieces of data about a file: either a local path to the file to be processed, or the file contents themselves parsed as a string. If both are provided, the parsed contents will be used." Working on decades old code has given me a new appreciation for good doc strings :)

Reply:
also, just `File` would be clear. then File.get_data() is real nice.

Reply:
The docstring already documents the parameters, I updated the description. I changed names to `InputFileData.get_contents()`